addon/components/mdc-tab-bar.js

```diff
@@ -35,11 +40,24 @@ export default Ember.Component.extend(MDCComponent, {
   layout,
   classNames: ['mdc-tab-bar'],
   classNameBindings: ['isIconsOnly:mdc-tab-bar--icon-tab-bar', 'isIconsWithText:mdc-tab-bar--icons-with-text', 'mdcClassNames'],
+  attributeBindings: ['style'],
   init() {
     this._super(...arguments);
     set(this, 'tabs', Ember.A([]));
     set(this, 'mdcIndicatorStyles', {});
   },
+  didInsertElement() {
+    this._super(...arguments);
+    if (get(this, 'scrolling')) {
```

Comment:
It seems like you might want to gate this based on the presence of the action instead of `scrolling`.